app/paypal.py

```
@@ -0,0 +1,181 @@
+    links: List[PayPalLinkModel]
+    event_version: str
+    resource_version: str
+
+async def login()->str:
+    async with aiohttp.ClientSession() as session:
+        async with session.get(f"https://{PAYPAL_ENDPOINT}/v1/oauth2/token", headers={
+            'Authorization': f"Basic {PAYPAL_BASE_AUTH}",
+            'Content-Type': 'application/x-www-form-urlencoded',
+        }, params=(
+            ('grant_type', 'client_credentials'),
+        )) as response:
```

Ruff (0.12.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)